ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-rhoam-addons__v4.13.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: rhods-rhoam-tests
```

Comment:
please rename the test
see https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/ and the naming of the team's tests

Reply:
is `mps-pipeline-rosa-rhods-rhoam` legitimate?

`MPS-Pipeline` is a suitable test marker for rhods-rhoam co-exist tests. 
The equivalence marker is `RHOAM-RHODS` which would have made duplication with the naming scheme 
..v4.13-[tests suit]-[platform]-[addons installed]:
`rhoam-rhoads-rosa-rhods-rhoam`

Should `mps-pipeline` replaced with `sanity`?

Reference:
https://github.com/red-hat-data-services/ods-ci/blob/478b497fbde17483b59c88b4dbb9faa9c15fe5ae/ods_ci/tests/Tests/600__ai_apps/640__rhoam/640__rhoam.robot#L23C15-L23C27